docs/SRIOV.md

```diff
@@ -0,0 +1,260 @@
+
+> Given the complexity involved in supporting Mellanox NICs, the Express tool will initially only support Intel NICs using the ixgbe driver. This includes the Intel X520, X540, and X550 families.
+
+The necessary configuration details can be implemented globally using **group_vars**, or on an individual host basis using **host_vars**. Each method is described below.
+
+### Host Variables
+
+Compute node-specific configurations can be implemented using what is known as host_vars. Configurations that may vary between hosts include:
+
+* Network interface name
+* Quantity of network interfaces used for SRIOV
+* Provider network mappings
+* Number of VFs per interface
+ 
+Using **host_vars**, the following are some variables that can be modified:
+
+* physical_device_mappings (required)
+* sriov_numvfs (required)
+* neutron_ovs_bridge_mappings (optional)
+
+In this example, two hosts have different NICs installed that report different names to the operating system.
+
+```
+root@compute01:~# ip link show
+...
+6: ens1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq portid 0002c90300ffe511 state UP mode DEFAULT group default qlen 1000
+    link/ether 00:02:c9:ff:e5:10 brd ff:ff:ff:ff:ff:ff
+7: ens1d1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq portid 0002c90300ffe512 state UP mode DEFAULT group default qlen 1000
+    link/ether 00:02:c9:ff:e5:11 brd ff:ff:ff:ff:ff:ff
+```
+
+```
+root@compute02:~# ip link show
+...
+3: ens1f0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000
+    link/ether 90:e2:ba:a2:1b:88 brd ff:ff:ff:ff:ff:ff
+5: ens1f1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000
+    link/ether 90:e2:ba:a2:1b:89 brd ff:ff:ff:ff:ff:ff
+```
+
+NIC naming can vary based on the kernel version, NIC driver, and the PCI slot where the card is installed. In this example, the NIC installed in each host is from a different manufacturer and uses a different driver:
+
+```
+root@compute01:~# ethtool -i ens1
+driver: mlx4_en
+version: 4.0-0
+firmware-version: 2.42.5000
+expansion-rom-version:
+bus-info: 0000:08:00.0
+supports-statistics: yes
+supports-test: yes
+supports-eeprom-access: no
+supports-register-dump: no
+supports-priv-flags: yes
+```
+
+```
+root@compute02:~# ethtool -i ens1f0
+driver: ixgbe
+version: 5.1.0-k
+firmware-version: 0x61bd0001
+expansion-rom-version:
+bus-info: 0000:08:00.0
+supports-statistics: yes
+supports-test: yes
+supports-eeprom-access: yes
+supports-register-dump: yes
+supports-priv-flags: yes
+```
+
+The **host_vars** for each host can be implemented in a file that corresponds to the host's short name located at **/opt/pf9-express/host_vars/<shortname>.yml**. In the following example, **compute01** uses a single network interface for SR-IOV, while **compute02** uses two. SR-IOV networks will leverage a new provider label named **sriov** and 8 VFs per interface, as shown here:
```

Comment:
Rename to hv01/hv02 to align with readme / sample inventory file

Reply:
fixed.